src/main/java/org/kohsuke/github/GHUser.java

```diff
@@ -231,10 +236,52 @@ public String getBio() {
      */
     public PagedIterable<GHGist> listGists() throws IOException {
         return root.createRequest()
-                .withUrlPath(String.format("/users/%s/gists", login))
+                .withUrlPath(format("/users/%s/gists", login))
                 .toIterable(GHGist[].class, null);
     }
 
+    /**
+     * Returns open projects of this user.
+     *
+     * <p>If the user represents an {@linkplain #getType() organization}, still returns its project list.
+     *
+     * @return the user project list
+     * @throws IOException
+     *         in case something goes wrong when fetching the project list
+     */
+    public PagedIterable<GHProject> listProjects() throws IOException {
+        return listProjects(GHProject.ProjectStateFilter.OPEN);
+    }
+
+    /**
+     * Returns projects of this user which have the passed {@code status}.
+     *
+     * <p>If the user represents an {@linkplain #getType() organization}, still returns its project list.
+     *
+     * @param status
+     *            the status filter
+     * @return the user project list
+     * @throws IOException
+     *         in case something goes wrong when fetching the project list
+     */
+    public PagedIterable<GHProject> listProjects(GHProject.ProjectStateFilter status) throws IOException {
+        return root.createRequest()
+                .withPreview(INERTIA)
+                .with("state", status)
+                .withUrlPath(projectsApiUrl())
+                .toIterable(GHProject[].class, item -> item.wrap(root));
+    }
```

Comment:
```suggestion
    /**
     * Returns projects of this user which have the passed {@code state}.
     *
     * <p>If the user represents an {@linkplain #getType() organization}, still returns its project list.
     *
     * @param state
     *            the state filter
     * @return the user project list
     * @throws IOException
     *         in case something goes wrong when fetching the project list
     */
    public PagedIterable<GHProject> listProjects(GHProject.ProjectStateFilter state) throws IOException {
        return root.createRequest()
                .withPreview(INERTIA)
                .with("state", state)
                .withUrlPath(projectsApiUrl())
                .toIterable(GHProject[].class, item -> item.wrap(root));
    }
```